NEWS.md

```diff
@@ -0,0 +1,11 @@
+# data.table.threads v1.0.0.
+
+## New Features
+- Added new arguments `verbose` and `time` to `runBenchmarks` and `findOptimalThreadCount`, allowing users to enable/disable detailed message printing and specify the number of benchmark repetitions.
```

Comment:
times instead of time?

Reply:
Thanks!
I updated that change in #9 (also, could you please review that PR and let me know if there is anything else I should implement or change for this package?)